skins/cat17/src/app/tests/reducers/test_form_content.js

```diff
@@ -82,6 +82,22 @@ test( 'When CHANGE_CONTENT sets membership type to active, address type is switc
 	t.end();
 } );
 
+test( 'When CHANGE_CONTENT sets payment type to sofort and payment interval is at least monthly, interval is set to once', function ( t ) {
```

Comment:
I agree, but the problem is that the whole directory full of tests uses this pattern and I think it's not bad enough to go through all files and fix them, I think a simple search and replace will also not do it because they describe different content changes. 

🤷‍♂️